src/Pages/Home/components/ListOfAlerts.tsx

```diff
@@ -1,61 +1,72 @@
+import { useTranslation } from "react-i18next"
 import { AlertSvg } from "../../../components/Icons/Alert";
 import { AlertIssueSvg } from "../../../components/Icons/AlertIssueSvg";
 import { WarnSvg } from "../../../components/Icons/WarnSvg";
+import { MuteAlertSvg } from "../../../components/Icons/MuteAlert";
+
+type Props = {
+    isAlertMuted: boolean,
+    setIsAlertMuted: () => void;
+}
+export function ListOfAlerts({ isAlertMuted, setIsAlertMuted }: Props) {
+    const { t } = useTranslation();
 
-export function ListOfAlerts() {
     return <div className="flex flex-col gap-2 p-2 h-[80vh]">
         {/* This is the opened controlled menu */}
-        <div className="flex justify-between ">
-            <div className="flex gap-2 bg-secondary p-1 rounded">
-                <div className=" bg-active text-active-foreground p-1 rounded">not read</div>
-                <div className="p-1 rounded">archive</div>
+        <div className="flex justify-between text-sm text-center">
+            <div className="flex gap-1 bg-secondary p-1 rounded-[20px]">
```

Comment:
it is better for us to use relative units

Reply:
Done.
See the PR in the internal network.